homeassistant/components/notify/aws_lambda.py

```diff
@@ -23,13 +25,11 @@
 CONF_PROFILE_NAME = "profile_name"
 CONF_CONTEXT = "context"
 
-PLATFORM_SCHEMA = vol.Schema({
-    vol.Required(CONF_PLATFORM): "aws_lambda",
-    vol.Optional(CONF_NAME): vol.Coerce(str),
-    vol.Optional(CONF_REGION, default="us-east-1"): vol.Coerce(str),
-    vol.Inclusive(CONF_ACCESS_KEY_ID, "credentials"): vol.Coerce(str),
-    vol.Inclusive(CONF_SECRET_ACCESS_KEY, "credentials"): vol.Coerce(str),
-    vol.Exclusive(CONF_PROFILE_NAME, "credentials"): vol.Coerce(str),
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_REGION, default="us-east-1"): cv.string,
```

Comment:
Should we keep name in here to have multiple notifications of the same platform?